sympy/printing/pretty/stringpict.py

```diff
@@ -33,12 +33,14 @@ def __init__(self, s, baseline=0):
         self.baseline = baseline
         self.binding = None
 
+    _remove_combining = dict.fromkeys(list(range(768, 880)) + list(range(8400, 8433)))
```

Comment:
It would probably be better to add a `line_width` function in `pretty_symbology.py` and use it here. It would be better to spell the numbers as `ord(c)` where `c` is one of the characters as used in `is_combining`.

Reply:
> It would probably be better to add a `line_width` function in `pretty_symbology.py` and use it here. 

  
Do you mean deleting the function inside the class `stringPict` and use the new `line_width()` defined in `pretty_symbology.py` instead? 


Sorry for the late reply, I was busy with my college assignments.

Reply:
The `stringPict` method could just call the function from `pretty_symbology`.

Reply:
> t would be better to spell the numbers as `ord(c)` where `c` is one of the characters as used in `is_combining`.

I'm afraid I don't get it. `is_combining()` takes characters as input, why would it need a new `ord()` function?

Reply:
I mean that rather than using opaque integers like `768` we should use the characters by their unicode names as is done in `is_combining` e.g. `ord('\N{COMBINING GRAVE ACCENT}')`.

Reply:
My bad. I didn't realize you were referring to the Python `ord()` function.